src/frontend/components/message-settings.js

```diff
@@ -37,7 +37,8 @@ export class MessageSettings extends LitElement {
         <message-menu-option
           iconName=${IconNames.mdiContentCopy}
           text="Copia"
-          @click=${() => this.copyToClipboard(this.message.content)}
+          @click=${() =>
+            this.copyToClipboard(this.message.content.replaceAll("\\\n", "\n"))}
```

Comment:
la replace sarebbe meglio metterla in una funzione apposita che ne isola il comportamento
valutare poi se ha senso richiamarla dentro copyToClipboard